main.go

```diff
@@ -84,7 +89,7 @@ func main() {
 
 		if linesWritten < *lineLimit {
 			// Keep writing to the same file, increment counter
-			if _, err := fmt.Fprintln(currentFile, scanner.Text()); err != nil {
+			if _, err := fmt.Fprintf(currentFile, fmt.Sprintf("%s%s", scanner.Text(), lineEnding)); err != nil {
```

Comment:
So my question is, should this be optional? Or do you think this is something that everyone running this on Windows would want?

Reply:
I think this is something everyone would want.  When evaluating a program like this, the first thing someone would do is to recombine all of the parts back together to make sure the program is working properly.  Without CRLF, they will notice the file size of the original and recombined file to be different.  I think it is a good *piece of mind* to have CRLF as default for Windows as it is then easier to verify the correctness of `file-splitter`.

Another option would be to read in the first line (by opening the input file in binary mode), determine the current line ending, and then output using the same style line ending.

Reply:
Want to go for the --crlf flag then? :) I fear that forcing CRLF wouldn't come in handy if someone is trying to do some cross-OS stuff, say splitting a file on windows but sending it to a unix OS.

Reply:
I think adding --crlf would be good. You can just add it in like you did the other flags.